stdio/examples/stdio.rs

```diff
@@ -0,0 +1,13 @@
+extern crate jsonrpc_stdio_server;
+
+use jsonrpc_stdio_server::server;
+use jsonrpc_stdio_server::jsonrpc_core::*;
+
+fn main() {
+  let mut io = IoHandler::default();
+  io.add_method("say_hello", |_params| {
+    Ok(Value::String("hello".to_owned()))
+  });
+
+  server(io);
```

Comment:
I would prefer a `ServerBuilder` to be inline with other transports.
(It would be super simple though, just `SeverBuilder::new(io).build()`)